### ISSUE TITLE
Pterodactyl Service Config

### ISSUE THREAD

**Issue by lostinsanity** [VertisanPRO/pterodactyl#476]
### Describe the issue

When trying to update a package that sues the pterodactyl service config we are constrained to the same constraints as the variables the egg has. So if an egg is configured correctly and the port variable is configured as required|integer|between:1024,65535 I am unable to configure the port as AUTO_PORT as previously instructed. I am also unable to leave required variables such as a steam username blank for the user to fill in and must now put a placeholder for everything which gets quite time consuming. Especially considering the new pterodactyl service changes and that I need to touch every package we have to enable the console and all of that.

### Product Version

2.1.0

### Pterodactyl Panel Version (if applicable)

1.11.7

### To Reproduce

1. Log into your wemx panel as admin
2. Go to Products & Services > Packages
3. Select a Pterodactyl package using a properly configured egg that has variable restrictions in place
4. Make any changes and try to save.

### Expected Behaviour

The admin panel for package creation should not be tied to the same constraints as the server itself is. Or I have to change every egg in use to mak things not required which would break the service or change it in a way a user can unintentionally break it.

**Comment by GIGABAIT93** [comment]
Hello, is this problem still relevant?

**Comment by lostinsanity** [comment]
Yes it is, I am attaching two screenshots. One is of the service on Wemx where I am trying to set AUTO_PORT and the other is a capture of the variable on pterodactyl that is configured as an integer. In this situation I am unable to configure AUTO_PORT as the field is set as an integer for a port number which is the proper restriction to ensure it can't be changed incorrectly. I am aware this is where permissions come into play and is how I currently prevent users from being able to change this, but to do so I have to manually edit the variable in pterodactyl to configure it to a string which I think might be possible to change the Wemx restriction on that if it matches the place holder variables provided?
![ptero-example](https://github.com/user-attachments/assets/5c7e4a8d-1868-4b3e-95b7-ea2a3f41b217)
![wemx-example](https://github.com/user-attachments/assets/c3203e2e-d6a6-4432-8beb-0cc6b9156b61)

**Comment by GIGABAIT93** [comment]
I can disable variable checking in the package settings, but that can introduce other risks if the admin doesn't configure something correctly, then it will be difficult for us to determine the cause of the problem if the server is not created at the time of purchase

**Comment by lostinsanity** [comment]
Is it possible to just add the approved and usable placeholders as acceptable variables regardless of the configured variable type or is that getting too into the weeds?

**Comment by GIGABAIT93** [comment]
I will think about how this problem can be solved.

**Comment by lostinsanity** [comment]
Understood, if it can't be done so in an easy way that make sense, could I just recommend putting additional text on the Wemx service side under under the acceptable placeholders highlighting that the pterodactyl egg will need to be configured to remove integer restrictions for use of the place holders in case any others run into the same issue? Or possibly some documentation aded to the Wemx docs for that?

**Comment by GIGABAIT93** [comment]
![зображення](https://github.com/user-attachments/assets/0f28c03e-efa5-44d9-b29f-85714f96d304)
i can add that. Since I came to the conclusion that disabling all validation rules breaks everything and there are no more buttons or selects, everything looks like a text input
